tools/benchmarking.sh

```diff
@@ -1,13 +1,21 @@
 #!/usr/bin/env bash
 
-# This script can be used for running moonbeam's benchmarks.
+# This script can be used for running tanssi's benchmarks.
 #
-# The moonbeam binary is required to be compiled with --features=runtime-benchmarks
+# The tamsso binary is required to be compiled with --features=runtime-benchmarks
 # in release mode.
 
 set -e
 
-BINARY="./target/production/tanssi-node"
+# By default we use the tanssi-node release binary
+# However we can use any binary by running the benchmark tool with
+# BINARY=./target/release/container-chain-template-simple-node ./tools/benchmarking.sh
+if [[ -z "${BINARY}" ]]; then
+    BINARY="./target/release/tanssi-node"
```

Comment:
The default was production instead of release, why this change?

Reply:
I figured it's easier for us to use in this case to make sure benchmarks run correctly. We can set production in the benchmarking server only